pkg/store/consul.go

```diff
@@ -127,9 +127,9 @@ func (s *Consul) GetRange(prefix string) (values [][]byte, err error) {
 		if pair.Key == prefix {
 			continue
 		}
-		values = append(values, pair.Value)
+		kvi = append(kvi, &kviTuple{pair.Key, pair.Value, pair.ModifyIndex})
```

Comment:
I'd put a more meaningful name to the array of `kviTuple` like `entries`/`tuples` or `values` instead of `kvi`. Same thing about other methods.